Makefile

```diff
@@ -154,6 +154,8 @@ docker-build: ## Build docker image with the manager.
 	eval $$(minikube docker-env --profile kgw); docker build \
 		--tag ${MANAGER_IMG} \
 		--tag kusk-gateway:latest \
+		--tag kubeshop/kusk-gateway:latest \
+		--tag kubeshop/kusk-gateway:$(shell git describe --tags $(shell git rev-list --tags --max-count=1)) \
```

Comment:
Without this line I cannot get `make create-env` to pickup my local changes.

See: <https://github.com/kubeshop/kusk-gateway/pull/746/files#r987136390>.

One of these lines is having an effect on my local changes being built or not.